libs/aws/langchain_aws/chat_models/bedrock_converse.py

```diff
@@ -406,26 +407,30 @@ def set_disable_streaming(cls, values: Dict) -> Any:
             model_parts[-2] if len(model_parts) > 1 else model_parts[0]
         )
 
-        # As of 09/15/24 Anthropic and Cohere models support streamed tool calling
+        # As of 12/03/24 Anthropic, Cohere and Amazon Nova models support streamed tool calling
         if "disable_streaming" not in values:
             values["disable_streaming"] = (
                 False
                 if values["provider"] in ["anthropic", "cohere"]
                 else "tool_calling"
             )
+            if "nova" in model_id:
+                values["disable_streaming"] = False
```

Comment:
Setting `disable_streaming` to False means that users cannot selectively turn OFF streaming when using tool calls for Nova models. Is that the expected behavior? Here is the relevant documentation.
https://github.com/langchain-ai/langchain/blob/a220ee56cd5a1050fbb682c330251ffc3fe0eb09/libs/core/langchain_core/language_models/chat_models.py#L210

We should also add a check for the provider.
```suggestion
            if values["provider"] == "amazon" and "nova" in model_id:
                values["disable_streaming"] = False
```